README.md

```diff
@@ -1,138 +1,70 @@
-3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
-4. Push to the Branch (`git push origin feature/AmazingFeature`)
-5. Open a Pull Request
-
-<!-- MARKDOWN LINKS & IMAGES -->
-<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
-[contributors-shield]: https://img.shields.io/github/contributors/ffxivcode/AutoDuty.svg?style=for-the-badge
-[contributors-url]: https://github.com/ffxivcode/AutoDuty/graphs/contributors
-[forks-shield]: https://img.shields.io/github/forks/ffxivcode/AutoDuty.svg?style=for-the-badge
-[forks-url]: https://github.com/ffxivcode/AutoDuty/network/members
-[stars-shield]: https://img.shields.io/github/stars/ffxivcode/AutoDuty.svg?style=for-the-badge
-[stars-url]: https://github.com/ffxivcode/AutoDuty/stargazers
-[issues-shield]: https://img.shields.io/github/issues/ffxivcode/AutoDuty.svg?style=for-the-badge
-[issues-url]: https://github.com/ffxivcode/AutoDuty/issues
+# AutoDuty
+
+<img align="right" width="150" height="150" src="logo.png">
+
+AutoDuty (AD) is a Dalamud plugin for FFXIV that serves as a tool to assist in the creation and following of paths through dungeons and duties. It's designed to enable automatic looping of instanced content with Duty Support, Trusts, or Squadrons.
+
+Support for it can be found in the [Puni.sh Discord server](https://discord.gg/punishxiv).
+
+_Unlicensed code, which follows [default copyright law](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license)_
+
+# Dungeon Paths
+
+<img align="right" height="250" src="Assets/paths.png">
+
+Each dungeon supported by AutoDuty has a pre-configured path laid out, which should complete the dungeon without any issues. If any issues are encountered, make sure to reach out in the [support Discord](https://discord.gg/punishxiv).
+
+The paths will also run over all loot containers and, when configured, will pick up the loot and do something with it while either in between dungeon runs or after your set amount of runs are over.
+
+Creating paths is also supported and easily accomplished within AutoDuty by using the `Build` tab at the top. For path status, please refer to the [pinned spreadsheet](https://discord.com/channels/1001823907193552978/1236757595738476725/1243059104528994334) in the AutoDuty channel of the [support Discord](https://discord.gg/punishxiv).
+
+# Automatic Leveling
+
+<img align="right" height="250" src="Assets/leveling.png">
+
+AutoDuty has a leveling feature that will continuously run you through the "correct" dungeon to level through. The dungeon is typically determined by path quality, support for boss fights, and obviously the level of the dungeon.
+
+With AutoDuty, you can level a job from 15 to the max level very quickly and fully AFK, provided you have the correct gear and the option in the AD configuration to automatically the best gear.
```

Comment:
equip?

Reply:
Fixed in [2e3e495](https://github.com/ffxivcode/AutoDuty/pull/317/commits/2e3e49574234e05830d2c685c7d4677bdcb37b43)